.circleci/config.yml

```diff
@@ -1,11 +1,10 @@
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:8.11.3
+    - image: circleci/node:14.15.4
```

Comment:
not sure if this is a problemo, I am looking to it

Reply:
not a problem (as long as the build still runs)

Reply:
thank you @tonybajan-globality really appreaciate!